### PR TITLE
test generator

### PR DESCRIPTION
**Pull request by saiqulhaq**
PR for https://github.com/shakacode/shakapacker/issues/34

is it ok if the generator app uses RSpec instead of Minitest?

### CODE REVIEW

**Comment by tomdracz** [comment]
Me and @justin808 have been briefly chatting about moving to RSpec so that part is not a problem I think, ideally would do that before/in parallel with this one so we end up with single test suite

**Comment by justin808** [comment]
@saiqulhaq keep us posted! this looks promising.

**Comment by justin808** [comment]
@saiqulhaq I just kicked off CI. I'm truly excited to see how this one goes!

**Comment by justin808** [comment]
@saiqulhaq any updates?

**Comment by saiqulhaq-hh** [comment]
TLDR: Hi sorry I have no passion to complete this PR

I've shared this gem with the community
but this is one of their responses https://github.com/decidim/decidim/discussions/8783#discussioncomment-2104076

**Comment by saiqulhaq-hh** [comment]
you may close this PR

**Comment by justin808** [comment]
Any volunteers interested in continuing this PR?

**Comment by ahangarha** [comment]
> can you pick this one up?

@justin808 I like to work with generators and tests. Yet I am not sure if I am ready to take this task for this project. If you think so, I work on it.

File: rakelib/examples.rake [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    desc "Clobbers (deletes) #{example_type.name_pretty}"
+    task example_type.clobber_task_name_short do
+      rm_rf(example_type.dir)
+    end
+
+    # GENERATE
+    desc "Generates #{example_type.name_pretty}"
+    task example_type.gen_task_name_short => example_type.clobber_task_name do
+      mkdir_p(example_type.dir)
+      sh_in_dir(examples_dir, "rails new #{example_type.name} #{example_type.rails_options} --skip-javascript -B")
```
Comment (justin808) [inline rakelib/examples.rake:34]: @ahangarha check this out.

**Pull request closed without merging**